### DISCUSSION TITLE
Unable to get watchPaths working.

### DISCUSSION THREAD

**Discussion by KasparRosin** [remix-run/remix#3896] [Q&A] [answered]
Here is how my remix config looks like.
```js
exports.watchPaths = async () => {
  return ['/ui/*'];
};
```

Here is how my folder structure looks like
```
│ remix.config.js
│
└───app
│   │   ...
│   
└───ui (git submodule)
    └───components
    └───styles
```

When changing files inside ui/ nothing changes.
I also tried to console log inside the watchPaths function, but nothing was logged.

Edit: 
Attempted variants
```
'./ui/**/*'
'/ui/**/*'
```

**Comment by kiliman** [comment, marked as answer]
The paths must be _relative_ to the project root. You should use `./ui/**/*` so it also watches subfolders of `ui`.

Also, you cannot export `watchPaths` directly. It must be inside the config object, like this. It appears the documentation is incorrect.
```js
module.exports = {
  ...
  watchPaths: ['./ui/**/*']
}
```
<img width="927" alt="image" src="https://user-images.githubusercontent.com/47168/182162515-6c76cb2b-a526-4147-a57a-caa6784bb338.png">

**Reply by KasparRosin** [reply]
Did not work. 😞

**Reply by KasparRosin** [reply]
Updated the original post with attempted variants

**Reply by KasparRosin** [reply]
@kiliman Thanks for the update.
I think something might be wrong in my setup then. Couldn't get any of the examples working. Even tried creating a new folder for testing.

**Reply by kiliman** [reply]
Which version of Remix are you on? `watchPaths` was added in v1.6.4.

**Reply by KasparRosin** [reply]
Ahh! Thank you!
I'm on version 1.6.3 🤦🏻
